alignak/misc/common.py

```diff
@@ -45,9 +45,9 @@
 """
 from collections import namedtuple
 try:
```

Comment:
It's an old code before some times ago, setproctitle not in requirements, so now we can import it without use try